### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by nnsW3**
This PR addresses several typographical errors across various files in the project. The changes improve readability and maintain the professional standard of the documentation and code comments.

Justification
Typographical errors, while minor, can detract from the overall quality of the project. Correcting these errors ensures clarity and professionalism, making the project more accessible and understandable for current and future contributors.

Best regards.

### CODE REVIEW

File: docs/docs/api/api/puffer-client-helpers.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ configuration
 
 `object`
 
-The public client created with viem.
+The public client was created with viem.
```
Comment (9inpachi) [inline docs/docs/api/api/puffer-client-helpers.md:38]: This was correct. Please revert this change. And API docs are generated from comments in the code, so updating it here won't work.

**Review by 9inpachi — CHANGES_REQUESTED** [review]
Thanks for contributing!

I have added one comment.

File: docs/docs/api/api/puffer-client-helpers.md (hunk 1 above)
Comment (nnsW3) [reply docs/docs/api/api/puffer-client-helpers.md]: done

**Review by 9inpachi — APPROVED** [review]
